src/routes/manage_job/get.rs

```diff
@@ -0,0 +1,156 @@
+use crate::errors::ReacherError;
+use crate::routes::check_email::header::check_header;
+use sqlx::{Pool, Postgres};
+use warp::Filter;
+
+use serde::Serialize;
+
+use check_if_email_exists::CheckEmailOutput;
+use sqlx::types::chrono::{DateTime, Utc};
+use sqlx::types::Json;
+
+/// NOTE: Type conversions from postgres to rust types
+/// are according to the table given by
+/// [sqlx here](https://docs.rs/sqlx/latest/sqlx/postgres/types/index.html)
+#[derive(Debug, Serialize, PartialEq, Eq)]
+pub enum ValidStatus {
+	Running,
+	Completed,
+	Stopped,
+}
+
+impl Default for ValidStatus {
+	fn default() -> Self {
+		ValidStatus::Running
+	}
+}
+
+/// Job record stores the information about a submitted job
+///
+/// `job_status` field is an update on read field. It's
+/// status will be derived from counting number of
+/// completed email verification tasks. It will be updated
+/// with the most recent status of the job.
+#[derive(sqlx::FromRow, Debug, Serialize)]
+pub struct JobRecord {
+	id: i32,
+	created_at: DateTime<Utc>,
+	total_records: i32,
+}
+
+/// Email record stores the result of a completed email verification task
+///
+/// It related to it's parent job through job_id
+/// It stores the result of a verification as jsonb field
+/// serialized from `CheckEmailOutput`
+#[derive(sqlx::FromRow, Debug)]
+pub struct EmailRecord {
```

Comment:
This struct is not used in the code, right?

I propose to remove it, unless there's some sqlx best practices to keep the data models of your tables?

Reply:
Yeah I'll remove it.